routes/api/orderAPI.js

```diff
@@ -0,0 +1,95 @@
+// Gravity Application Server | orderAPI
+// =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+module.exports = function (express) {
+  var router = express.Router();
```

Comment:
that is in es6, not es5. 'const' notation won't work until they recieve browser support hopefully in 2016. make sure your on the es5 airbnb style guide.

https://github.com/airbnb/javascript/tree/master/es5
not
https://github.com/airbnb/javascript